source/vocab/details.ttl

```diff
@@ -943,21 +943,21 @@
 
 :otherYear a owl:DatatypeProperty;
     sdo:domainIncludes :PrimaryProvisionActivity ;
-    rdfs:label "kompletterande datum"@sv;
+    rdfs:label "other year"@en, "kompletterande datum"@sv;
```

Comment:
Should be "kompletterande årtal" in Swedish?

Reply:
Could be! Added a note about it for now. I try to keep off content-changes which might require updated helptexts :)

Reply:
Fair enough!